myuw/models/__init__.py

```diff
@@ -340,3 +327,39 @@ class Meta(object):
             ["user", "section_label"],
         ]
         ordering = ['section_label']
+
+
+class MigrationPreference(models.Model):
+    user = models.OneToOneField('User', on_delete=models.CASCADE)
```

Comment:
Would it be good to add an index on this field?

Reply:
models.OneToOneField implies unique=True, this means Django will automatically create an index for this field

Reply:
Interesting, thank you!